src/components/10-atoms/fieldset/index.react.d.ts

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 
 export interface AXAFieldsetProps {
-  horizontal?: boolean;
+  horizontal?: string;
```

Comment:
Instead of string you could write the 2 strings that are allowed in that way: `'string' | 'horizontal'`.

Or like the button d.ts as a extra TS type: `type Size = 'small' | 'large';`

Reply:
Done🙌🏼